src/test/java/uk/gov/pay/directdebit/payments/services/DirectDebitEventServiceTest.java

```diff
@@ -224,6 +225,19 @@ public void registerMandatePendingEventFor_shouldCreateExpectedEvent() {
         assertThat(directDebitEvent.getEventDate(), is(ZonedDateTimeMatchers.within(10, ChronoUnit.SECONDS, ZonedDateTime.now())));
     }
 
+    @Test
+    public void registerMandateCreatedEventFor_shouldCreateExpectedEvent() {
+        service.registerMandateCreatedEventFor(mandateFixture.toEntity());
+
+        verify(mockedDirectDebitEventDao).insert(prCaptor.capture());
```

Comment:
I assume `prCaptor` is left over from when we had “payment requests”…

Reply:
organic growth by copy paste